samples/grid-pro/demo/exporting/demo.js

```diff
@@ -0,0 +1,40 @@
+const csvExport = document.querySelector('#csvExport');
+const jsonBtn = document.querySelector('#jsonExport');
+const csvDownload = document.querySelector('#csvDownload');
+const jsonDownload = document.querySelector('#jsonDownload');
+const result = document.querySelector('#result');
+const modifiedDataToggle = document.querySelector('#modifiedDataToggle');
+
+const grid = Grid.grid('container', {
+    dataTable: {
+        columns: {
+            product: ['Apples', 'Pears', 'Plums', 'Bananas'],
+            weight: [100, 40, 0.5, 200],
+            price: [1.5, 2.53, 5, 4.5],
+            metaData: ['a', 'b', 'c', 'd']
+        }
+    },
+    columnDefaults: {
+        cells: {
+            editMode: {
+                enabled: true
+            }
+        }
```

Comment:
```suggestion
        }
        filtering: {
            enabled: true
        }
```
Since we mention filtering in the description